### PR TITLE
Marcus v1.3b ug

### PR DESCRIPTION
**Pull request by marcusleeeugene**

### CODE REVIEW

**Review by Prabhakaran-Gokul — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -203,11 +205,22 @@ Parameter: none
...
+Opens up a window, showing the existing details of an entry and allows for quick corrections and updates without 
+requiring the use of prefixes and command syntax in the UI. 
+
+Use `TAB` key to iterate through fields, `Ctrl + S` (Windows) or `Command + S` (Mac) to save, `ESC` key to quickly 
+exit the revise window.
+
+Note:
+- Leave a white-space between categories to separate them
+- Leave a white-space between schools to separate them
+- Leave a blank line to differentiate between each reviews
```
Comment (Donavanty) [inline docs/UserGuide.md:223]: Good additiona

File: docs/UserGuide.md [hunk 2]
```diff
@@ -219,6 +232,27 @@ Example:
 
     revise 1
 
+![Revise Window UI](images/ReviseWindowUi.png)
```
Comment (Donavanty) [inline docs/UserGuide.md:235]: Makes more sense

**Review by Donavanty — APPROVED** [review]
LGTM

**Pull request merged**